lnwallet/commitment.go

```diff
@@ -229,6 +229,18 @@ func CommitScriptToRemote(chanType channeldb.ChannelType, csvTimeout uint32,
 	}, nil
 }
 
+func InitiatorFee(chanType channeldb.ChannelType, commitWeight int64,
```

Comment:
Would there be a better term for `miner_fee + anchors` then "fee"? To prevent introducing a second definition of fee. Cost, overhead, ..?

Reply:
I like `cost` perhaps?